assets/js/modules/analytics/components/dashboard/DashboardUniqueVisitorsWidget.stories.js

```diff
@@ -0,0 +1,167 @@
+/**
+ * DashboardUniqueVisitorsWidget Component Stories.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { CORE_USER } from '../../../../googlesitekit/datastore/user/constants';
+import { STORE_NAME } from '../../datastore/constants';
+import { provideModules, provideSiteInfo } from '../../../../../../tests/js/utils';
+import { provideAnalyticsMockReport } from '../../util/data-mock';
+import { withWidgetSlug } from '../../../../googlesitekit/widgets/util/';
+import WidgetReportError from '../../../../googlesitekit/widgets/components/WidgetReportError';
+import WidgetReportZero from '../../../../googlesitekit/widgets/components/WidgetReportZero';
+import WithRegistrySetup from '../../../../../../tests/js/WithRegistrySetup';
+import DashboardUniqueVisitorsWidget, { selectSparklineArgs, selectReportArgs } from './DashboardUniqueVisitorsWidget';
+
+const currentEntityURL = 'https://www.example.com/example-page/';
+
+const Template = ( args ) => {
+	const widgetSlug = 'dashboardUniqueVisitorsWidget';
+
+	return (
+		<WithRegistrySetup func={ args.setupRegistry }>
+			<div className="googlesitekit-widget">
+				<div className="googlesitekit-widget__body">
+					<DashboardUniqueVisitorsWidget
+						WidgetReportError={ withWidgetSlug( widgetSlug )( WidgetReportError ) }
+						WidgetReportZero={ withWidgetSlug( widgetSlug )( WidgetReportZero ) }
+					/>
+				</div>
+			</div>
+		</WithRegistrySetup>
+	);
+};
+
+export const Ready = Template.bind();
```

Comment:
Storybook docs show `.bind` always called with `{}` although it doesn't explain why (the best I could find was [this](https://stackoverflow.com/questions/65054041/how-does-technique-for-making-a-copy-of-a-function-works-in-javascript-storyboo)). This corresponds to the `thisArg` parameter in the function so my guess is that it matters for class-based components. While we don't really use these anymore, there are cases where it is unavoidable (e.g. error boundaries). We also use this in the stories for `Buttons` now as well so we should be consistent and use it everywhere.

Reply:
👍